src/client/ldBrowser.tsx

```diff
@@ -1,39 +1,32 @@
-import { Component, ReactNode } from 'react';
-import { initialize, LDFlagChangeset, LDFlagSet } from 'launchdarkly-js-client-sdk';
+import { ReactNode, useState } from 'react';
+import { initialize, LDClient, LDFlagChangeset, LDFlagSet } from 'launchdarkly-js-client-sdk';
 
 import { LDReactContext as HocState, Provider } from '../shared/context';
 import { getFlattenedFlagsFromChangeset } from '../shared/utils';
 
 type LDBrowserProps = { children: ReactNode };
 
-class LDBrowser extends Component<LDBrowserProps, HocState> {
-  readonly state: Readonly<HocState>;
+let ldClient: LDClient;
```

Comment:
Declared outside of scope so that this variable is render-stable?

I bet there's a "react" way to do this within the main function, perhaps with a ref? But if not or if it's not apparent, this is fine.